src/GettingStarted.md

```diff
@@ -38,31 +25,37 @@ Give your repo a name and click on "Create repository":
 <img src="img/my-3pi.png" alt="new repo"/>
 
 ## Clone your repository
-On the command line on your host machine, clone the repo.
-For an SSH-based setup, run:
-
-```bash
-$ git clone git@github.com/<username>/<reponame>.git
-```
-
-Or, for a setup that uses GitHub CLI, run:
+On the command line on your host machine, change directory to the location where you would like to check out your repository. Let us assume that you named your repo `my-3pi`. Check it out using the following command (where `<username>` must be substituted with your GitHub username):
 
 ```bash
 $ gh repo clone <username>/<reponame>
 ```
 
+This will create a directory called `my-3pi` in the current working directory.
+
 The template includes [raspberrypi/pico-sdk](https://github.com/raspberrypi/pico-sdk) as a submodule, which itself also has a lot of submodules. We recommend against using the `--recursive` flag because we do not need to recursively clone the submodules inside of `pico-sdk`. Instead, change directory into the root of your clone and run:
```

Comment:
"Instead, change directory into the root of your clone and run the submodule update:"
The way it's written makes it seem like they should change directory and then cd again.

Reply:
Good catch! I just removed the extra `cd` from the code listing...